bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
+    placement_options.add_argument('--assignment_method', help='Taxonomic assignment method, either pplacer (phylogenetic) or DIAMOND (pairwise). default = pplacer', default=Run.PPLACER_TAXONOMIC_ASSIGNMENT, choices=(Run.PPLACER_TAXONOMIC_ASSIGNMENT,Run.DIAMOND_TAXONOMIC_ASSIGNMENT))
     
     pplacer_options = graft_parser.add_argument_group('pplacer assignment options')
-    pplacer_options.add_argument('--placements_cutoff', metavar='confidence', help='Cutoff of placement confidence level (0.5 - 1), default=0.75', default=0.75)
-    pplacer_options.add_argument('--resolve_placements', action="store_true", help='Force taxonomy down to the best placement (default=False)', default=False)
-    pplacer_options.add_argument('--no_merge_reads', action="store_true", help='Merge forward and reverse read alignments for placement (default=False)', default=False)
+    pplacer_options.add_argument('--placements_cutoff', metavar='confidence', help='This flag allows you to change the likelihood cutoff for phylogenetic placement of reads (default=0.75).', default=0.75)
+    pplacer_options.add_argument('--resolve_placements', action="store_true", help='Ignore the placements cutoff and simply use the best placement assigned to the read. (default=False)', default=False)
+    pplacer_options.add_argument('--no_merge_reads', action="store_true", help='When this flag is specified, The alignment of the forward and reverse reads will be merged before placement. This provides more information for phylogenetic placement, and the classifications provided by pplacer generally improve, so try to use this flag when you can. (default=False)', default=False)
 
     nucleotide_options = graft_parser.add_argument_group('nucleotide search-specific options')
-    nucleotide_options.add_argument('--search_hmm_files', nargs='+', help='Specify .hmm files to use in search step', default=argparse.SUPPRESS)
-    nucleotide_options.add_argument('--search_hmm_list_file', metavar='newline-separated file with paths to each search HMM', default=argparse.SUPPRESS)
-    nucleotide_options.add_argument('--search_diamond_file', nargs=1, help='Specify .dmnd file to use in search step', default=None)
-    nucleotide_options.add_argument('--aln_hmm_file', help='Specify a single .hmm file to use in align step (default: the search_hmm_file if there is only 1)', default=argparse.SUPPRESS)
-    nucleotide_options.add_argument('--euk_hmm_file', help='Specify the .hmm file to use in the check for euk contamination', default=argparse.SUPPRESS)
+    nucleotide_options.add_argument('--search_hmm_files', nargs='+', help='This flag is to specify the HMM(s) used to search the data with - note you cannot change the HMM that is used to align the sequences. This is because alignments must match the alignment length of the sequences used in the placement tree, otherwise the placement step will fail. If providing more than one HMM, separate with a space.', default=argparse.SUPPRESS)
```

Comment:
Well, you can specify it with `--aln_hmm_file`, though. Maybe this extra detail belongs to that flag?